lighthouse-core/gather/gather-runner.js

```diff
@@ -491,14 +501,21 @@ class GatherRunner {
 
   static instantiateGatherers(passes, rootPath) {
```

Comment:
let's do instantiateGatherers back in config so by the time `gatherrunner.run` is called, everything has an instance. 
this'll require updating some tests. sorry 'bout that.

Reply:
on it